src/handler.rs

```diff
@@ -281,7 +281,7 @@ where
     A::Context: AsyncContext<A>,
 {
     fn handle<R: ResponseChannel<M>>(self, ctx: &mut A::Context, tx: Option<R>) {
-        ctx.spawn(self.then(move |res, this, _| {
+        ctx.spawn(Box::pin(Pin::from(self)).then(move |res, this, _| {
```

Comment:
Looks like this type is a little more complex than it have to. We need a `Sized` and `Unpin` `ActorFuture`, the current suggestions has type:
`Pin<Box<Pin<Box<dyn ActorFuture<Output = T, Actor = A>>>>>`
The most external pin seams unnecessary, something like this should suffice:
`Box<Pin<Box<dyn ActorFuture<Output = T, Actor = A>>>>`

If I'm not missing anything this could be `Box::new(Pin::from(self))` instead, which I think makes code a tiny bit easier to read.